server/planning/templates/event_download_default.html

```diff
@@ -0,0 +1,3 @@
+{% for item in items %}{% if item.get('place') %}{{ item.place | join(', ') }}{{"\r\n"}}{% endif %}{% if item.formatted_state %}({{ item.formatted_state.upper() }}) {% endif %}{{ item.get('name') }}{% if item.get('definition_short') %}: {{ item.definition_short }}{% endif %}{{ "\r\n" }}Schedule: {{ schedule }}{{ "\r\n" }}{% if item.formatted_location %}Location: {{ item.formatted_location }}{{ "\r\n" }}{% endif %}{% for contact in item.contacts %}Media contact: {{ contact }}{{ "\r\n" }}{% endfor %}
```

Comment:
The schedule is no longer rendering in with template.

Change `{{ schedule }}` to `{{ item.schedule }}`